Doc/library/collections.rst

```diff
@@ -224,7 +224,7 @@ For example::
 .. class:: Counter([iterable-or-mapping])
 
     A :class:`Counter` is a :class:`dict` subclass for counting hashable objects.
-    It is an unordered collection where elements are stored as dictionary keys
+    It is a collection where elements are stored as dictionary keys
```

Comment:
I found more occurrences of «arbitrary order» and usage of `sorted` to make examples deterministic in the rest of the file.  I think careful thought should be given to the guarantees given in this file: it may be easy to say that `elements` returns in insertion order, but `most_common` may need to stary under-defined.

If you could get the attention of @rhettinger on the bug tracker to see what’s the best thing to do here, or possibly to not touch this file in this PR, that would be best.

Reply:
Thanks for you great review, but I am not very understand your meaning about:

"but most_common may need to stary under-defined.".

Could you explain it a little more?

Reply:
Fixed the typo!

The basic idea is that if we guarantee that all these methods return things in insertion order now, we may preclude future implementation changes or optimizations.  Each method needs to be checked to see if it’s useful to change the doc that says «unordered» (if it’s accurate now) or not (if we don’t want to guarantee it).

Reply:
@merwok How about creating another pull request about it?
This pull request is too long discussed.

Reply:
I think that if a file is updated, it should be updated exhaustively, not add more ambiguity than was before.  But I don’t want to block the updates to the tutorial, so feel free to merge when you are satisfied.